File: tier1/php/5c2ecc56/store.php
<?php

function lookup($code) {
    $link = mysqli_connect('localhost', 'app', '', 'orders');
    $statement = mysqli_prepare($link, 'SELECT status FROM orders WHERE code = ?');
    mysqli_stmt_bind_param($statement, 's', $code);
    mysqli_stmt_execute($statement);
    return mysqli_stmt_get_result($statement);
}

File: tier1/csharp/9508f4a3/Store.cs
using Microsoft.Data.SqlClient;

namespace App;

public static class Store
{
    public static object Lookup(string code)
    {
        var statement = "SELECT status FROM orders WHERE code = '" + code + "'";
        using var command = new SqlCommand(statement);
        return command.ExecuteScalar();
    }
}
